packages/oauth/oauth-provider/src/assets/asset.ts

```diff
@@ -1,9 +1,8 @@
 import type { Readable } from 'node:stream'
+import type { ManifestItem } from '@atproto-labs/rollup-plugin-bundle-manifest'
 
 export type Asset = {
   url: string
-  type?: string
-  isEntry: boolean
-  sha256: string
+  item: ManifestItem
```

Comment:
The `AssetRef` abstraction used in other parts of the code no longer need an `sha256` (was incorrectly used in CSP code). Exposing the full `ManifestItem` here allows for more control (e.g. generating `<link rel="preload" />` of scripts that are dynamically imported).